### ISSUE TITLE
RStudio Desktop - boost libraries incompatibility on openSUSE

### ISSUE THREAD

**Issue by usuallycwdillon** [rstudio/rstudio#9920]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.4.1717-65.1
    OS Version      : openSUSE Tumbleweed
    R Version       : 4.1.1

### Steps to reproduce the problem
Tumbleweed is currently using boost libraries at 1.77, but 1.76 is required for RStudio.

Attempting to install RStudio from command line:

```
Problem: nothing provides 'libboost_filesystem.so.1.76.0()(64bit)' needed by the to be installed rstudio-1.4.1717-65.1.x86_64
 Solution 1: do not install rstudio-1.4.1717-65.1.x86_64
 Solution 2: break rstudio-1.4.1717-65.1.x86_64 by ignoring some of its dependencies
```

### Describe the problem in detail
Rstudio currently cannot be installed on Tumbleweed. This seems to be a repeat of the issue just prior to RStudio release 1.4.1103, when boost 1.74 was required over 1.75.  

### Describe the behavior you expected
I expected that RStudio can be installed on Tumbleweed.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
Note: Proposing for no later than Spotted Wakerobin, to coincide with June release of openSUSE 15.4 which could potentially run into the same issue.

**Comment by mikebessuille** [comment]
Related to https://github.com/rstudio/rstudio/issues/9854

**Comment by gsgxnet** [comment]
I am  using RStudio dailies since a long time on Tumbleweed, at this moment: 
`RStudio 2022.01.0-daily+199 "Prairie Trillium" Daily (686bc16e95ead41172a6b5ab9726a12007514c16, 2021-11-11) for OpenSUSE 15`  
On my system, both 1.67 and 1.77 boost libs are installed ... go figure. 

The dailies get me into other issues though. See #9854 

I do hope both issues will be resolved soon.

**Comment by jgutman** [comment]
@ronblum As of 2022.06.0+167, the bundled version of Boost used by RStudio has been upgraded to 1.78.0. It is unclear if this will resolve this particular issue on OpenSUSE, or if additional work is needed.

**Comment by ronblum** [comment]
Will it work on the current version, openSUSE 15.3?

**Comment by jgutman** [comment]
I don't know, I was hoping someone with an OpenSUSE system could test it!

**Comment by jgutman** [comment]
It does work on our daily builds for OpenSUSE 15, but I'm not sure of the exact version number used

**Comment by usuallycwdillon** [comment]
RStudio opens and seems to run well on Tumbleweed, but there is an occasional blink. I haven't done a proper upgrade in a while, so there could be other things causing that. I'll do a full `zypper dup` within an hour and get back on that issue.

**Comment by gsgxnet** [comment]
Final remark for `OpenSUSE` easy installs for manual downloads. I have simply defined a local folder as a repository source with an elevated priority. Into that I download the newest RStudio builds. 
Also add the right RStudio signature keys into the package management.  
Thereafter the normal `zypper up` (or `zypper dup` for Tumbleweed) updates will pick up these manual downloads in their next runs.  
Only once, when RStudio changed its naming  scheme, I had to do the next update manually.
